ts/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -65,7 +65,6 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
             await this.fcJobInfoCollector.retrieveTrialStatus(this.kubernetesCRDClient);
             if (this.kubernetesJobRestServer.getErrorMessage !== undefined) {
                 throw new Error(this.kubernetesJobRestServer.getErrorMessage);
-                this.stopping = true;
```

Comment:
Seems better to move this in front of the `throw new Error`.

Reply:
Hello J-shang, I thought about this at the beginning, but than I realized `this.stopping` never been assigned to `true` in the old logic, so if I move `this.stopping=true` to the front of `throw new Error`, `this.stopping` will be `true` when throwing Error and the old logic will be broken.